pygame/compat.py

```diff
@@ -114,3 +114,26 @@ def next_(i, *args):
     import itertools.imap as imap_
 except ImportError:
     imap_ = map
+
+
+# The behaviour of the power operator, '**' and built-in pow() function,
+# changed in Python 3.x.  In Python 3.x raising a negative number to a
+# fractional power results in a complex number, while in earlier versions
+# it raised a ValueError.  Matching the behaviour of pygame requires
+# the ValueError instead of a complex result.  The compatibility function
+# defined below is provided for this purpose.
```

Comment:
While math.pow will return a ValueError, it is also using the underlying C math library and will have a different precision from pow.

pygame is explicitly using pow underneath it's implementation, so if we use math.pow(), there will be differences between the implementations. Whether these will matter is another question, since they'll arguably only show up in odd corner cases.